tests/idem/structs.rs

```diff
@@ -14,6 +14,18 @@ pub struct Foo {
 
 struct Bar;
 
+struct NewInt<T: Copy>(i32, SomeType /* inline comment */, T);
+
+struct Qux<'a,
+           N: Clone + 'a,
+           E: Clone + 'a,
+           G: Labeller<'a, N, E> + GraphWalk<'a, N, E>,
+           W: Write + Copy>
+    (AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA, // Comment
+     BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB,
+     G,
+     W);
```

Comment:
I think I would format like this:

```
struct Qux<'a,
           N: Clone + 'a,
           E: Clone + 'a,
           G: Labeller<'a, N, E> + GraphWalk<'a, N, E>,
           W: Write + Copy>(
    AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA, // Comment
    BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB,
    G,
    W,
);
```

Note the trailing comma - we probably want three options for trailing commas in tuples - always, never, and vertical only (maybe we have these options already, actually).